swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -7380,7 +7541,19 @@ components:
       example: 
         {
         "scaRedirect": {"href": "https://www.testbank.com/asdfasdfasdf"},
-        "self": {"href": "/v1/payments/sepa-credit-transfers/1234-wertiq-983"}
+        "self": {"href": "/psd2/v1/payments/sepa-credit-transfers/1234-wertiq-983"}
+        }
+
+    _linksPaymentInitiationStatus: 
+      description: |
+        Should refer to next steps if the problem can be resolved via the interface e.g. for re-submission of credentials.
+      type: object
+      additionalProperties: 
+        $ref: "#/components/schemas/hrefType"
+      example: 
+        {
+        "scaRedirect": {"href": "https://www.testbank.com/asdfasdfasdf"},
+        "self": {"href": "/psd2/v1/payments/sepa-credit-transfers/1234-wertiq-983"}
```

Comment:
Relevant Change:  _linksPaymentInitiationStatus is new
Adoption approach: Follow the change